packages/mux-audio-react/src/index.tsx

```diff
@@ -66,18 +58,17 @@ const MuxAudio = React.forwardRef<HTMLAudioElement | undefined, Partial<Props>>(
       playerSoftwareVersion,
       autoplay: autoPlay,
     };
-    const nextPlaybackEngineRef = initialize(propsWithState, mediaElRef.current, playbackEngineRef.current);
-    playbackEngineRef.current = nextPlaybackEngineRef;
-    setUpdateAutoplay(() => {
-      if (!mediaElRef.current) return;
-      setupAutoplay(mediaElRef.current, autoPlay, playbackEngineRef.current);
-    });
```

Comment:
to have 2 of these was a too noisy, returning a plain object from initialize makes things a lot easier IMO